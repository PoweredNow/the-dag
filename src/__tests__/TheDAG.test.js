const { TheDAG } = require('../');

describe('TheDAG', () => {
  test('exports', () => {
    expect(TheDAG).toMatchSnapshot();
  });
  test('is instantiatable', () => {
    const aDAG = new TheDAG({ nodes: [], edges: [] });
    expect(aDAG).toMatchSnapshot();
  });
});
