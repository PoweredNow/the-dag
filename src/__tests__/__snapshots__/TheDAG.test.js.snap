// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheDAG exports 1`] = `[Function]`;

exports[`TheDAG is instantiatable 1`] = `TheDAG {}`;
